client/src/plus/collections/collection-list-item.tsx

```diff
@@ -30,23 +29,27 @@ export function CollectionListItem({
   const iconLabel = _getIconLabel(item.url);
 
   return (
-    <article key={item.id} className="icon-card">
```

Comment:
This must have a `key` property and this just uncovers really bad HTML. This is:
```html
<ul>
  <div>
    <article></article>
    <article></article>
    <article></article>
  </div>
</ul>
```
Could you change this to a proper list?

Reply:
Changed into an actual list, but the key property should sit outside the component, see: https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys